appserver/extras/docker-images/server-full/src/main/docker/Dockerfile

```diff
@@ -7,77 +7,31 @@ FROM @docker.java.image@
 # 8181: https
 EXPOSE 4848 9009 8080 8181
 
-# Initialize the configurable environment variables
-ENV HOME_DIR=/opt/payara\
-    PAYARA_DIR=/opt/payara/appserver\
-    SCRIPT_DIR=/opt/payara/scripts\
-    CONFIG_DIR=/opt/payara/config\
-    DEPLOY_DIR=/opt/payara/deployments\
-    PASSWORD_FILE=/opt/payara/passwordFile\
-    # Payara Server Domain options
-    DOMAIN_NAME=@docker.payara.domainName@\
-    ADMIN_USER=admin\
-    ADMIN_PASSWORD=admin \
-    # Utility environment variables
-    JVM_ARGS=\
-    PAYARA_ARGS=\
-    DEPLOY_PROPS=\
-    POSTBOOT_COMMANDS=/opt/payara/config/post-boot-commands.asadmin\
-    PREBOOT_COMMANDS=/opt/payara/config/pre-boot-commands.asadmin
-ENV PATH="${PATH}:${PAYARA_DIR}/bin"
+ENV DOMAIN_NAME=@docker.payara.domainName@ \
+    PAYARA_ARGS="" \
+    PREBOOT_COMMANDS=${CONFIG_DIR}/pre-boot-commands.asadmin \
+    POSTBOOT_COMMANDS=${CONFIG_DIR}/post-boot-commands.asadmin \
+    DEPLOY_PROPS=""
 
-ARG TINI_VERSION=v0.18.0
-
-# Download tini
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini \
-    https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /
-
-RUN true \
-    # Install dependencies
-    && apt-get update \
-    && apt-get install -y coreutils gpg \
-    && rm -rf /var/lib/apt/lists/* \
-    # Create and set the Payara user and working directory owned by the new user
-    && groupadd --gid 1000 payara \
-    && useradd -u 1000 -M -s /bin/bash -d ${HOME_DIR} payara -g payara \
-    && echo payara:payara | chpasswd \
-    && mkdir -p ${DEPLOY_DIR} \
-    && mkdir -p ${CONFIG_DIR} \
-    && mkdir -p ${SCRIPT_DIR} \
-    && chown -R payara: ${HOME_DIR} \
-    # Verify tini
-    && gpg --batch --keyserver "hkp://ipv4.pool.sks-keyservers.net" --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 \
-    && gpg --batch --verify /tini.asc /tini \
-    && chmod +x /tini \
-    && true
-
-USER payara
-WORKDIR ${HOME_DIR}
-
-COPY --chown=payara:payara maven/artifacts/payara5 ${PAYARA_DIR}/
+COPY --chown=payara:payara maven/artifacts/@docker.payara.rootDirectoryName@ ${PAYARA_DIR}/
```

Comment:
Revert order of these 2 COPY commands as script content will change less likely as Payara Server. -> reuse of layers